packages/dropdownable/components/Dropdownable.tsx

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import styled, { css } from "react-emotion";
+
+import Overlay from "../../shared/components/Overlay";
+import DropdownContents from "./DropdownContents";
+
+export enum Direction {
```

Comment:
We'll probably need to support a dropdown that's centered relative to it's parent's width